multiversx_sdk_cli/tests/test_cli_wallet.py

```diff
@@ -298,6 +298,74 @@ def test_verify_not_signed_message(capsys: Any):
     assert all(word in out for word in text)
 
 
+def test_sign_and_verify_message_with_multi_address_pem(capsys: Any):
+    multi_address_pem_path = testdata_path / "multiple_addresses.pem"
+    message = "test"
+
+    return_code = main(["wallet", "sign-message", "--message", message, "--pem", str(multi_address_pem_path), "--pem-index", "0"])
+    out = json.loads(_read_stdout(capsys))
+
+    assert False if return_code else True
+    assert out == {
+        "address": "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th",
+        "message": "test",
+        "signature": "0x7aff43cd6e3d880a65033bf0a1b16274854fd7dfa9fe5faa7fa9a665ee851afd4c449310f5f1697d348e42d1819eaef69080e33e7652d7393521ed50d7427a0e"
+    }
+
+    return_code = main(["wallet", "verify-message",
+                        "--address", "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th",
+                        "--message", message,
+                        "--signature", "0x7aff43cd6e3d880a65033bf0a1b16274854fd7dfa9fe5faa7fa9a665ee851afd4c449310f5f1697d348e42d1819eaef69080e33e7652d7393521ed50d7427a0e"
+                        ])
+    assert False if return_code else True
+
+    out = _read_stdout(capsys)
+    text = """SUCCESS: The message "test" was signed by erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th""".split()
+    assert all(word in out for word in text)
```

Comment:
Just comparing `out` and `text` wouldn't have worked?

Reply:
Not really, because we display the message using the `show_message()` function that uses `rich` for printing.